docs/mappingservice-crd-reference.md

```diff
@@ -0,0 +1,93 @@
+# MappingService Custom Resource Reference
+
+## Simple CR Example
+
+```yaml
+apiVersion: saas.3scale.net/v1alpha1
+kind: MappingService
+metadata:
+  name: simple-example
+spec:
+  image:
+    tag: mapping-service-v3.8.0-r3
+  replicas: 2
+  env:
+    apiHost: http://system:3000
+    previewBaseDomain: stg-saas.3scale.net
+  secret:
+    systemAdminTokenVaultPath: secret/data/openshift/stg-saas-ocp/3scale/mappingservice-system-master-access-token
```

Comment:
We are using fake paths on all our VaultPath, something like `secret/data/openshift/cluster-example/3scale/mappingservice-system-master-access-token`

Can you please update all referenced real possible paths (simple, full, and cr example)?

Reply:
6a8e13e